homeassistant/components/fully_kiosk/config_flow.py

```diff
@@ -39,6 +45,8 @@ async def _create_entry(
             host,
             DEFAULT_PORT,
             user_input[CONF_PASSWORD],
+            use_ssl=user_input[CONF_SSL],
+            verify_ssl=user_input[CONF_VERIFY_SSL],
```

Comment:
If the SSL settings are incorrect, what error will be raised when calling `fully.getDeviceInfo` below?

Reply:
If `use_ssl` is wrong, meaning that it is set to `True` but the Fully app is not configured for SSL, the error will be "Failed to connect" (cannot_connect with the translations folder removed) regardless of the state of `verify_ssl.`

If `use_ssl` is correct, meaning that it is set to `True` and the Fully app is configured for SSL; but `verify_ssl` is wrong, meaning that HA has not been loaded with the CA to verify the certificate, there will be an SSL error from the backend library.

Reply:
> there will be an SSL error from the backend library

OK, and how is that shown to the user?

Reply:
On the frontend the user gets "Failed to connect" as caught by 
```suggestion
            except (
            ClientConnectorError,
            FullyKioskError,
            asyncio.TimeoutError,
        ) as  error:
```
![image](https://user-images.githubusercontent.com/19595786/225700835-4dc708f9-0a73-43cd-8010-8b16f178419b.png)

And the debug log registers:
aiohttp.client_exceptions.ClientConnectorCertificateError: Cannot connect to host 192.168.1.30:2323 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)')]

Reply:
OK, in that case I'd suggest to set the error to something more detailed than `cannot_connect` to help the user distinguish between the errors.

Reply:
That was my intention originally but there's nothing for ssl errors in homeassistant/strings.json. I guess the general expectation is that the user will check the logs for the specific failure when getting "Failed to connect", specially if he is advanced enough to be setting up ssl with the integration.

Reply:
How does the user see the info passed in `description_placeholders`? The frontend doesn't display it.

Reply:
Now the exception message is shown to the user on the frontend, as recommended.